### ISSUE TITLE
Time stamps on FeaturePhenotypeAssociation

### ISSUE THREAD

**Issue by sarahhunt** [ga4gh/g2p-team#13]
The FeaturePhenotypeAssociation record needs time stamps. The new convention appears to be recordCreateTime and recordUpdateTime - is that correct @mbaudis ?

**Comment by mbaudis** [comment]
Yes and no ;-) While there is a defined format/naming for time stamps (below), there seems to be resistance to include them in the (non-metadata) objects (keeping them out of the schema & letting this be part of the local data implementations, I guess).

IMHO at least for the annotation-rich objects one should use them.

```
    /**
    The time at which this record was created.
    Format: ISO 8601, YYYY-MM-DDTHH:MM:SS.SSS (e.g. 2015-02-10T00:03:42.123Z)
    */
    string recordCreateTime;

    /**
    The time at which this record was last updated.
    Format: ISO 8601, YYYY-MM-DDTHH:MM:SS.SSS (e.g. 2015-02-10T00:03:42.123Z)
    */
    string recordUpdateTime;
```

**Comment by diekhans** [comment]
I don't there is resistance. We are still working on get things
stabilized and synced with schemas/complaince/server/docs.

Patience my dear Watson ...

**Comment by sarahhunt** [comment]
Thanks! So that's the format we should use here and other parts of the API will catch up later.

**Comment by mbaudis** [comment]
I've added a note about _Common Attribute Names and Formats_ to the [metadata_intro.rst](https://github.com/ga4gh/schemas/blob/metadata/doc/source/metadata_intro.rst) document.

A [formatted version of this](http://wiki.ga4ghdata.org/Wiki/GA4GHdata/MetadataAPIintroDoc) can now be found at the ga4ghdata.org site.
